src/assets/svg/bridge-logo-large.svg

```diff
@@ -1,25 +1,11 @@
-<path d="M111.118 0H109.09L109.057 5.94519L111.084 6.04647V0H111.118Z" fill="#29707A"/>
-<path d="M121.964 10.0327C121.964 12.6337 120.748 14.1542 118.652 14.1542C116.557 14.1542 115.375 12.6344 115.375 10.0327C115.353 9.24749 115.491 8.46609 115.78 7.73553C115.104 7.63425 114.428 7.56651 113.752 7.49878C113.458 8.31092 113.309 9.16841 113.313 10.0321C113.313 13.444 115.34 15.6399 118.01 15.6399C118.837 15.6326 119.644 15.3863 120.333 14.9307C121.023 14.4751 121.566 13.8297 121.897 13.0724V15.8089C121.897 17.6332 120.545 18.5112 118.855 18.5112C117.371 18.5112 116.084 17.768 115.915 16.3179H113.853C114.09 18.7499 116.286 20.0001 118.855 20.0001C121.424 20.0001 123.889 18.7163 123.889 15.7773V9.42503C123.213 9.22247 122.537 9.05346 121.861 8.88444C121.931 9.26325 121.965 9.64761 121.964 10.0327Z" fill="#29707A"/>
-<path d="M118.653 5.90928C119.233 5.87859 119.808 6.01956 120.308 6.31449C120.807 6.60943 121.208 7.04521 121.461 7.56717C122.306 7.76973 123.117 7.97229 123.928 8.17549V4.62747H121.9V7.26397C121.644 6.43613 121.127 5.71296 120.427 5.20168C119.727 4.6904 118.881 4.41824 118.014 4.42556C117.294 4.42247 116.585 4.59653 115.948 4.93235C115.312 5.26818 114.768 5.75545 114.364 6.35116C115.108 6.41868 115.851 6.51997 116.595 6.655C116.869 6.39684 117.193 6.19749 117.547 6.06924C117.901 5.94099 118.278 5.88655 118.653 5.90928Z" fill="#29707A"/>
-<path d="M131.226 14.4578C129.401 14.4578 128.184 13.1741 128.015 10.8098C127.339 10.5395 126.663 10.3034 125.987 10.0667V10.1679C125.987 13.5121 127.914 15.9441 131.226 15.9441C131.918 15.982 132.61 15.8788 133.261 15.6407C133.912 15.4026 134.507 15.0346 135.011 14.5591C134.538 14.2211 134.031 13.8837 133.527 13.5456C133.234 13.8581 132.876 14.102 132.479 14.2597C132.081 14.4175 131.653 14.4851 131.226 14.4578V14.4578Z" fill="#29707A"/>
-<path d="M136.398 9.99867C136.398 6.58677 134.302 4.39087 131.227 4.39087C128.422 4.39087 126.564 6.18101 126.124 8.81621C127.799 9.36078 129.436 10.0153 131.024 10.7754H136.398C136.381 10.5167 136.381 10.2573 136.398 9.99867V9.99867ZM128.017 9.25552C128.254 7.0622 129.369 5.91136 131.193 5.91136C133.018 5.91136 134.202 7.26283 134.336 9.25552H128.017Z" fill="#29707A"/>
-<path d="M136.126 12.4653H134.064C134.623 12.7982 135.164 13.1591 135.686 13.5465C135.893 13.2136 136.042 12.848 136.126 12.4653V12.4653Z" fill="#29707A"/>
+<svg xmlns="http://www.w3.org/2000/svg" width="137" height="20" fill="none" viewBox="0 0 137 20">
```

Comment:
The size doesn't match what's in figma (156px). I also think we can maybe get rid of small/large and just have one bridge logo (it will scale).

Reply:
Updated svg's